compiler/test/dotc/pos-test-pickling.blacklist

```diff
@@ -42,6 +42,7 @@ i9999.scala
 13491.scala
 7512.scala
 i6505.scala
+i15155.scala
```

Comment:
Why adding this test to the blacklist? Isn't it supposed to pass after reverting the changes?

Reply:
There are many matchtype tests that fail pickling. Someone should find and fix the cause. Previously the test was a neg test, so no pickling was attempted.